CHANGELOG.rst

```diff
@@ -102,6 +103,7 @@
 
 .. _`traceback style docs`: https://pytest.org/latest/usage.html#modifying-python-traceback-printing
 
+.. _#469: https://github.com/pytest-dev/pytest/issues/469
```

Comment:
I realize this is the first time we are formatting the changelog like this, but I think this link would be better placed above the `2.9.0` header line, so the link targets reside at the bottom of their respective version headers.